--- avogadro-1.0.3/avogadro/src/CMakeLists.txt~	2011-04-25 05:22:18.000000000 +0000
+++ avogadro-1.0.3/avogadro/src/CMakeLists.txt	2015-06-27 11:08:27.021146247 +0000
@@ -83,7 +83,7 @@
 
 set_target_properties(avogadro-app PROPERTIES OUTPUT_NAME avogadro)
 
-target_link_libraries(avogadro-app ${OPENBABEL2_LIBRARIES} ${QT_LIBRARIES}
+target_link_libraries(avogadro-app ${OPENBABEL2_LIBRARIES} ${QT_LIBRARIES} X11
     avogadro)
 
 if(APPLE)
